src/Nordea/Components/Error.elm

```diff
@@ -0,0 +1,222 @@
+module Nordea.Components.Error exposing (..)
+
+import Css exposing (pct, rem, vh)
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes as Attributes exposing (href)
+import Nordea.Components.FlatLink as FlatLink
+import Nordea.Components.Text as Text
+import Nordea.Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Svg.Styled as Svg exposing (Svg)
+import Svg.Styled.Attributes as SvgAttrs exposing (d)
+
+
+type alias Locale =
+    { no : String, se : String, dk : String } -> String
+
+
+type alias ErrorConfig =
+    { errorType : ErrorType, locale : Locale }
+
+
+type ErrorType
+    = InternalServerError
+    | PageNotFound
+
+
+type Error msg
+    = Error ErrorConfig
+
+
+internalServerError : Locale -> Error msg
+internalServerError locale =
+    init { errorType = InternalServerError, locale = locale }
+
+
+pageNotFound : Locale -> Error msg
+pageNotFound locale =
+    init { errorType = PageNotFound, locale = locale }
+
+
+init : ErrorConfig -> Error msg
+init config =
+    Error { locale = config.locale, errorType = config.errorType }
+
+
+view : List (Attribute msg) -> List (Html msg) -> Error msg -> Html msg
+view attributes children (Error config) =
+    let
+        toI18NString textToTranslate =
+            config.locale textToTranslate
```

Comment:
Not sure I get this... 😅 Isn't this the same as just using `config.locale`? You define a function that just calls another function. If you want to rename the variable, you could at least do something like this:

```elm
toI18NString =
    config.locale
```

Right?

Reply:
You are completely right. Fixed it now!